community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/outputs.tf

```diff
@@ -55,6 +55,7 @@ output "nodeset_tpu" {
 }
 
 output "checksum" {
-  description = "Checksum of all files written to the bucket."
-  value       = local.checksum
```

Comment:
what about checksum for other files ?

Reply:
we don't need them. This output has been added in a first place, so we can order terraform resources creation, namely prevent controller startup before config is saved to GCS. After this change controller is "resilient enough" to  wait until config is created.